R/read.any.R

```diff
@@ -204,34 +215,93 @@ read.any <- function(
   }
 
   if(!is.null(widths) && is.null(col.names) || tolower(file_type) == 'fwf'){
-    x = read.fwf(filename, widths, header = FALSE)
+    x = utils::read.fwf(filename, widths, header = FALSE)
   }
 
   if(!is.null(widths) && !is.null(col.names) || tolower(file_type) == 'fwf'){
-    x = read.fwf(filename, widths, header = FALSE, col.names = col.names)
-  }  
- 
+    x = utils::read.fwf(filename, widths, header = FALSE, col.names = col.names)
+  }
+
+  #if(!is.null(sheets) && isexcel) {
+    
+   # if(sheets == 'ALL' || sheets == 'All' || sheets == 'all'){
+      
+    #  sheet_names = readxl::excel_sheets(filename)
+      
+    #} else if(is.numeric(sheets)) {
+      
+     # all_sheet_names = readxl::excel_sheets(filename)
+      #sheet_number_name_map <- setNames(all_sheet_names, seq_along(all_sheet_names))
+      #sheet_names = sheet_number_name_map[sheet_numbers]
+      
+    #} else {
+      
+     # sheet_names = sheets
+      
+    #}
+    
+    #sheet_data = lapply(sheet_names, function(sheet_name) {
+     # readxl::read_excel(filename, sheet = sheet_name)
+    #})
+    #x = setNames(sheet_data, sheet_names)
+    
+  #}  
```

Comment:
it seems this PR includes lots of work from a different issue? please resolve this. I'd suggest creating a separate fork for each issue.